tests/testthat/test_functions.R

```diff
@@ -0,0 +1,25 @@
+context("Tests of package functions")
+
+mock_data <- data.frame(
+  va = c(1:10),
+  vb = c(2:11),
+  vc = factor(c(rep(1:2, 5)), labels = c("one", "two")),
+  vd = c(rep(c("jeden", "dwa", "x", "y", "z"), 2)),
+  stringsAsFactors = FALSE)
+
+test_that("sum_columns returns vector of correct length", {
+  expect_length((sum_columns(mock_data)), 2)
+  expect_equal(sum_columns(mock_data)[[1]], 55)
```

Comment:
I'm not sure if this line checks the length of the output. I'd likely move it to a separate test named "sum_columns returns vector of correct length". It would be great if the created test will check values for all the columns of mock_data.

Reply:
i changed test description and moved column checks to another test